observability-lib/api/notification-policy.go

```diff
@@ -8,11 +8,32 @@ import (
 	"github.com/grafana/grafana-foundation-sdk/go/alerting"
 )
 
+func objectMatchersEqual(a alerting.ObjectMatchers, b alerting.ObjectMatchers) bool {
+	if len(a) != len(b) {
+		return false
+	}
+
+	for i := range a {
+		foundMatch := false
+		for j := range b {
+			if reflect.DeepEqual(a[i], b[j]) {
+				foundMatch = true
+				break
```

Comment:
this doesn't look like an exact match - could you leave a comment on the method explaining when a and b should be equal to each other?

Reply:
associated tests here : https://github.com/smartcontractkit/chainlink-common/pull/915/files#diff-5b4adecb60cf2e79cc22806a969468909ea9f8bfa52bd464a0add735ef682cbeR10-R46 it should explain what is expected

Reply:
agree Im adding the comment